static_src/components/stat.jsx

```diff
@@ -0,0 +1,52 @@
+
+import React from 'react';
+
+import style from 'cloudgov-style/css/cloudgov-style.css';
+
+import createStyler from '../util/create_styler';
+
+const STATES = [
+  'error',
+  'warning',
+  'info',
+  'success',
+  'none'
+];
+
+const propTypes = {
+  primaryStat: React.PropTypes.string.isRequired,
```

Comment:
Made these strings, so it's a pretty dumb layout component. This means ResourceUsage should take care of the state and such.